test_framework/scripts/longhorn-setup.sh

```diff
@@ -250,6 +275,7 @@ create_aws_secret(){
   yq e -i '.data.AWS_DEFAULT_REGION |= "'${AWS_DEFAULT_REGION_BASE64}'"' "${TF_VAR_tf_workspace}/templates/aws_cred_secrets.yml"
 
   kubectl apply -f "${TF_VAR_tf_workspace}/templates/aws_cred_secrets.yml"
+  kubectl apply -f "${TF_VAR_tf_workspace}/templates/aws_cred_secrets.yml" -n kube-system
```

Comment:
Why need to apply the same manifest twice in different namespaces?

Reply:
cluster-autoscaler is in kube-system, otherwise cluster-autoscaler is unable to see aws credentials.

Reply:
Well explained. The answer was in the example upstream manifests.